iOS-Email-Client/Libs/EventHandler.swift

```diff
@@ -24,13 +24,13 @@ class EventHandler {
 
     func handleEvents(events: [[String: Any]], completion: @escaping (_ result: EventData.Result) -> Void){
         var result = EventData.Result()
-        var successfulEvents = [Int32]()
+        var successfulEvents = [Any]()
```

Comment:
I thought the point of this was to have only one id in the `finishCallback` function